src/views/List.jsx

```diff
@@ -47,23 +49,32 @@ export function List({ listPath, data, listName, userIdFromPath }) {
 		<div>
 			{currentUserId && listName && <p>List name: {listName}</p>}
 			<label htmlFor="item-search"> Search for an item: </label>
-			<input
-				id="item-search"
-				type="text"
-				placeholder="Search item..."
-				onChange={handleChange}
-				value={searchItem}
-			/>
-			{searchItem && (
-				<button
-					type="button"
-					name="clearInput"
-					aria-label="Clear input"
-					onClick={() => setSearchItem('')}
-				>
-					X
-				</button>
-			)}
+			<Flex gap="3">
+				<Box width="22em">
+					<TextField.Root
+						variant="classic"
+						radius="large"
+						id="item-search"
+						onChange={handleChange}
+						value={searchItem}
+					>
+						<TextField.Slot>
+							<MagnifyingGlassIcon className="icon-styles" />
+						</TextField.Slot>
+					</TextField.Root>
+				</Box>
+				{searchItem && (
+					<IconButton
+						variant="solid"
+						type="button"
+						name="clearInput"
+						aria-label="Clear input"
+						onClick={() => setSearchItem('')}
+					>
```

Comment:
I know this isn't something that was handled on this issue, but I wanted to mention it because it's an easy fix to add here. I noticed that the clear button in light mode isn't an accessible color palette. The white against the blue background works great, but in light mode this one button in particular has black text, which does not meet the accessibility standards. Take a look at the screenshot: 
<img width="641" alt="Screenshot 2024-04-03 at 3 34 34 PM" src="https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/97368143/93fd0075-04ad-4de0-8941-9bb269c4c9eb">

Reply:
Thanks Christina! We are hoping that the light /dark mode issue will apply the themes that are accessible. This button is a Radix one, so the blue color should update once that merge is made.